modules/admin_manual/pages/configuration/files/manual_ file_ locking.adoc

```diff
@@ -0,0 +1,34 @@
+= Manual File Locking
+:toc: right
+
+== Introduction
+
+Starting with ownCloud Server 10.5 a new enhancement is provided for content collaboration. Manual file locking allows users to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out).
+
+The feature builds on the WebDAV Locks backend which has been introduced with Server 10.1 and is now available in the ownCloud Web Interface. Using the context menu of files, every user who has access can lock them. Users can recognize locked files by the means of a new lock indicator. While a file is locked, other users can still access it but they can not make any changes. Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).
+
+== Installation
+
+By default locks set in the web interface will expire after 30 minutes. The maximum lock time by default is 24 hours.
+
+NOTE: The user-facing components in the web interface are disabled by default. Administrators can enable the feature by executing the following occ command: 
+
+----
+occ config:app:set files enable_lock_file_action --value yes
+----
+
```

Comment:
```suggestion

**Make the user-facing components invisible**

[source,console,subs="attributes+"]
----
{occ-command-example-prefix} config:app:set files enable_lock_file_action --value no
----
```

Reply:
not needed. I think it's clear that if yes makes the component visible then no will make it invisible.

Reply:
incorrect, if there is a on/off variant, you have to describe both